### ISSUE TITLE
Some files are corrupted when I copy from local to weed mount directory

### ISSUE THREAD

**Issue by JaviCT** [chrislusf/seaweedfs#774]
**Describe the bug**
Almost all files are copied good but some of them are not completed. This is happening with the new release 1.08

**System Setup**
Ubuntu 18.04
Weed 1.08 amd64
Weed startup command:
`./weed -v=0 server -filer=true -dir=storage -pulseSeconds=1 -volume.images.fix.orientation=false`

Weed mount command:
`./weed mount -filer=localhost:8888 -dirListLimit=1000 -dir=./assets/media -filer.path=/some-folder`

Copy command:
`sudo cp -r assets/images assets/media`

**Expected behavior**
Before the update, I was working with the release v0.97 and the files were copied good, now some of them are corrupted.

**Screenshots**
Image to copy:
![1532944873](https://user-images.githubusercontent.com/19703794/48832469-21441f00-ed71-11e8-8781-bced7e8b2c38.jpg)

Image copied:
![1532944873](https://user-images.githubusercontent.com/19703794/48832490-2a34f080-ed71-11e8-840d-3e7a7c2c3612.jpg)

**Comment by chrislusf** [comment]
Is it always reproduceable? Need to reproduce. It'll be good to have the images. If not, the file paths and sizes.

**Comment by JaviCT** [comment]
Here is the link to the images that I am using:
https://drive.google.com/file/d/1WyTHQDNkhaK7WSirpsNAIWTPTxxjptVF/view?usp=sharing

**Comment by chrislusf** [comment]
Same problem if copied again?

**Comment by JaviCT** [comment]
Yes, I tried a few times and with different images

**Comment by chrislusf** [comment]
Thanks for all the details! Found the bug that was introduced 3 days ago. It is only in 1.08 for "weed mount".

**Comment by chrislusf** [comment]
released 1.09 with the fix

**Comment by JaviCT** [comment]
The new release is working fine for me, thanks @chrislusf

**Comment by ijunaid8989** [comment]
new release https://github.com/chrislusf/seaweedfs/releases/download/1.09/linux_amd64.tar.gz.. 

still says, 

`version 1.08 linux amd64`

**Comment by chrislusf** [comment]
Yeah. It's a manual process to change the version and I forgot to adjust
it. Need a way to automate this.

**Comment by Jevi06** [comment]
Hi @chrislusf ,I encoutered  the same problem in some of my nodes.
#### System Info
`OS:      CentOS Linux release 7.7.1908 (Core)`
`Kernel: Linux 3.10.0-957.el7.x86_64`
`Fuse: fuse-2.9.2-11.el7.x86_64`

#### How to reproduce 
In my cluster,there are many nodes with different OS version and kernel verion. the bug is reproduceable in above configuation.

#### seaweedfs version
I test this bug with seaweedfs  version 1.5 and 1.70.

I found no error info in seaweedfs log ,But if you need some other info that may help to conque this bug,just leave me  a message.

**Comment by chrislusf** [comment]
@Jevi06 There has been a lot of improvement for `weed mount` recently. There are no known bugs for FUSE in 1.70. Please make it reproducible if you can. A docker image would be ideal.

**Comment by Jevi06** [comment]
> ,I encoutered the same problem in some of my nodes.

Some test result here:
- Not all files corrupted when do copy operation.
- Videos always corrupted in my test.
- After mount filer to some point in my local filesystem tree,
   cp some/video/not/in/filer path/under/buckets `md5 changed,bug size keep the same`
   cp path/under/buckets some/video/not/in/filer  'md5 changed,size changed'

**Comment by Jevi06** [comment]
> n. A docker image would be ideal.

the image I use is pull from docker hub,the full signature is  `chrislusf/seaweedfs`,but for mounting,I install a fuse which version is `fuse-2.9.8-r2 x86_64 {fuse} (GPL-2.0-only LGPL-2.1-only)` in it.

**Comment by chrislusf** [comment]
Could you please create a new issue with details similar to this?  https://github.com/chrislusf/seaweedfs/issues/1182

The bug seems a bit complicated, file format, file size, fuse version, etc. All may have some effect on this.

**Comment by Jevi06** [comment]
> Could you please create a new issue with detailed steps similar to this? #1182
> 
> The bug seems a bit complicated, file format, file size, fuse version, etc. All may have some effect on this.

ok ,I  make a issue with more detail steps now.
